File: 4th/Java/src/D0926/Book.java
package D0926;
class Book
{
	private String title;
	private String author;
	
	public Book(String title, String author)
	{
		this.title = title;
		this.author = author;
	}
	
	public Book(String title)
	{
		this(title, "작자미상");
	}
	
	public Book()
	{
		this("", "");
	}
	
	public void show()
	{
		System.out.println(this.title + ", " + this.author);
	}
}

File: 6th/DB/P1108/bookDB/Book.cs
using System;
using System.Collections.Generic;

namespace P1108.bookDB;

public partial class Book
{
    public int Id { get; set; }

    public string Title { get; set; } = null!;

    public string Author { get; set; } = null!;

    public string Company { get; set; } = null!;

    public virtual ICollection<BookMember> BookMembers { get; set; } = new List<BookMember>();
}
